src/lib/response-triggers/help.ts

```diff
@@ -236,7 +250,47 @@ export default class Help extends Trigger {
 	createRoleConfigHelpMessage(message: MessageObject) : string {
 		if (this.config.checkRole(message.personId, Role.Admin) || this.config.checkRole(message.personId, Role.Config)) {
 			return `### How to configure role config values 🗞️!
-1. To get/edit/refresh/delete user roles, type \`@${constants.botName} roleconfig ${Object.values(ConfigAction).join('|')} {@User} {role}\`
+1. To get/edit/refresh/delete user roles, type \`@${constants.botName} ${Command.RoleConfig} ${Object.values(ConfigAction).join('|')} {@User} {role}\`
+1. I will respond in the room you messaged me in.`;
+		} else {
+			return this.createDefaultHelpMessage();
+		}
+	}
+
+	createLocationHelpMessage(message: MessageObject) : string {
+		if (this.config.checkRole(message.personId, Role.Admin) || this.config.checkRole(message.personId, Role.Location)) {
+			return `### How to configure location values!
+1. To get/edit/delete locations, type \`@${constants.botName} ${Command.Location} ${Object.values(LocationAction).join('|')} {location}\`
+1. I will respond in the room you messaged me in.`;
+		} else {
+			return `### How to get location values!
+1. To get a list of locations, type \'@${constants.botName} ${Command.Location} ${LocationAction.Get}\`
+    * To configure locations, please ask an admin.
+1. I will respond in the room you messaged me in.`
+		}
+	}
+
+	createUserLocationHelpMessage(message: MessageObject) : string {
+		if (this.config.checkRole(message.personId, Role.Admin) || this.config.checkRole(message.personId, Role.UserLocation)) {
+			return `### How to configure user location values!
+1. To get user locations for yourself or others, type \'@${constants.botName} ${Command.UserLocation} ${LocationAction.Get} me|all|unset|@User\`
+1. To set user locations, type \'@${constants.botName} ${Command.UserLocation} ${LocationAction.Set} {location} me|@User1 @User2\`
```

Comment:
So you'd write `@PockyBot location set brisbane @Laura @Evelyne me @Gillian`? And it would set all those people to Brisbane?

Reply:
yep, that's correct

Reply:
Ooh actually, me would have to be in a separate command as it is at the moment. It's either all mentions, or 'me'